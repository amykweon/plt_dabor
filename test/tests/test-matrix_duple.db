matrix[2][2]x;
duple y;

x = matrix_create([[1,2],[3,4]]);

y = (1,1);
print_int x[y];
x[y] = 14;
print_int x[y];

y = (1,0);
print_int x[y];
x[y] = 13;
print_int x[y];

y = (0,1);
print_int x[y];
x[y] = 12;
print_int x[y];

y = (0,0);
print_int x[y];
x[y] = 11;
print_int x[y];