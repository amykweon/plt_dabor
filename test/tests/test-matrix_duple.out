4
14
3
13
2
12
1
11